packages/ui-components/package.json

```diff
@@ -56,7 +57,8 @@
     "eslint-plugin-react-hooks": "^4.3.0",
     "react": "^16.14.0",
     "react-dom": "^16.8.0",
-    "typescript": "~4.7.4"
+    "typescript": "~4.7.4",
+    "@babel/plugin-proposal-private-property-in-object": "^7.21.11"
```

Comment:
ditto, resolutions

Reply:
Hmm, that's just a notice. Not an error. I agree we should try to make it go away but I think this solution is worse than the problem it solves.

We'd be better off keeping this notice, to remind us to us off `babel-preset-react-app` which isn't maintained.

Reply:
Fair enough, changed back